website/content/docs/configuration/credential-management/configure-credential-brokering.mdx

```diff
@@ -0,0 +1,67 @@
+---
+layout: docs
+page_title: Configure targets with credential brokering
+description: |-
+  How to configure targets with credential brokering
+---
+
+# Configure targets with credential brokering
+
+The following section provides steps to configure your targets with credential brokering.
+When you use credential brokering, Boundary centrally manages credentials and returns them to the user when they attempt to connect to a target.
```

Comment:
Since static cred stores can be managed by Boundary or Vault, I'd prefer this language

```suggestion
The following section provides steps to configure your targets with credential brokering.
When you use credential brokering, Boundary returns credentials to the user when they successfully connect to a target.
```